packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -423,6 +432,19 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
         }
     }
 
+    /**
+     * Resolves when the command is successfully sent, this doesn't mean that it
+     * was evaluated. Might reject if terminal wasn't properly started yet.
+     *
+     * Note that this method will try to escape your arguments as if it was
+     * someone inputting everything in a shell.
+     *
+     * Supported shells: `bash`, `cmd.exe`, `wsl.exe`, `pwsh/powershell.exe`
+     */
+    async executeCommand(commandOptions: CommandLineOptions): Promise<void> {
```

Comment:
docs should be aligned in  `TerminalWidget` and `TerminalWidgetImpl`

Reply:
ah true, will fix.